packages/kit/package.json

```diff
@@ -10,6 +10,7 @@
 	"type": "module",
 	"dependencies": {
 		"@sveltejs/vite-plugin-svelte": "^1.0.0-next.30",
+		"@vitejs/plugin-legacy": "^1.6.2",
```

Comment:
Could this become an optional dependency? 
That would keep the installation fast for non legacy projects